game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt

```diff
@@ -56,7 +56,7 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "max_mana_increase"                               "10 20 30 40 50 60"
+        "max_mana_increase"                               "20 30 40 50 60 75"
```

Comment:
Please don't change this. Its like this on purpose because ult passive doesn't work with our custom arcane orb.

Reply:
I only increased this to reflect the change of the mana cost nerf.

Reply:
how about
so:  10 14 19 24 60 75
(10% of the manacost, IE. You are gaining back 10% of the mana you used and this percentage changes obv when u get mana reduction items. At max level you'd get 75 mana back for 500 mana cost usage. pretty good but not broken or not complete garbage)